website/siteConfig.js

```diff
@@ -85,6 +85,12 @@ const siteConfig = {
       md.renderer.rules.button_open = custom_rules.button_open
       md.renderer.rules.button_close = custom_rules.button_close
 
+      // style diagram annotation links
+      // e.g. ((A))
+	custom_rules.keyRule(md,"annotate","\\(","\\)","annotate_open","annotate_close")
```

Comment:
I think that we should limit this to 1 character long @garethnunns did you remove that feature?

Reply:
I didn't remove the feature - the maximum number of characters allowed in the rule is currently 40 - I could add an argument for this.

It's also worth noting that the rule wouldn't work as described in the comment... 
That rule would match `\(like this\)` **not** `((like this))`

To make it work as described this should be changed to
````js
custom_rules.keyRule(md,"annotate","((","))","annotate_open","annotate_close")
````

Would be easier to fix if it were in a branch with just one feature in...

Reply:
Hi, sorry for lumping it all together. The comment was wrong, I initially tried ((A)) but thoguht \(A\) would be better, but I forgot to change the comment.